stats: record search latencies (#8868)
sourcegraph/sourcegraph: CHANGELOG.md

  [migration step](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/migrate.md) when upgrading
  past commit [821032e2ee45f21f701](https://github.com/sourcegraph/deploy-sourcegraph/commit/821032e2ee45f21f701caac624e4f090c59fd259) or when upgrading to 3.14.
  New installations starting with the mentioned commit or with 3.14 do not need this migration step.

> - Search statistics now include aggregated search latencies (in ms) of search queries, sent via pings.

Reviewer: Were other search metrics actually being sent in pings previously? Not sure how to read "now include" in this line.

Author: From the pings page:
`Whether a code search has ever been executed (true/false)`

`Aggregate counts of current users by product feature (site management, code search and navigation, code review, saved searches, diff searches)`

These are search statistics. But the `now` here doesn't really add much. What do you think of this:

```suggestion
- Log aggregated search latencies (in ms) of search queries, sent via pings.
```
This would go under `Added` in the Changelog, and not changed.

Reviewer: The verb "log" here still seems a bit imprecise to me. What do you think about:

```suggestion
- Aggregated search latencies (in ms) of search queries are now included in [pings](https://docs.sourcegraph.com/admin/pings).
```

Author: Works for me!
